### ISSUE TITLE
Sort order tag implementation?

### ISSUE THREAD

**Issue by prophetarmed** [saecki/mp4ameta#20]
Hi,

I love this library, fantastic for my music player I'm building at the moment. One feature I wish it were missing were implementations of the sort order tags, more specifically artist sort ("soar") and album artist sort ("soal"). 

Thank you for reading :)

**Comment by Saecki** [comment]
Yeah that seems like a desirable feature. I'll implement it after I get chapter parsing working.

**Comment by Saecki** [comment]
These are the sort order fields I could find:
- `soal` album sort order
- `soaa` album artist sort order
- `soar` artist sort order
- `sonm` title sort order
- `soco` composer sort order
- `sosn` show sort

is there anything missing?

**Comment by inalone** [comment]
I think those are all the ones I'm aware of, I can't think of what else there could be. I appreciate the quick response, thank you.

**Comment by saecki** [comment]
Hey I'm currently working again on this crate :) If you're still interested could you tell me if the PR looks right to you? I'm not sure if fields like artist sort order should also have accessors for multiple values, or just for a single one 🤔
